Tests/StreamChatTests/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -2987,6 +2987,8 @@ final class ChannelController_Tests: XCTestCase {
         controller.sendKeystrokeEvent {
             XCTAssertNil($0)
         }
+        wait(for: [env.eventSender!.keystroke_completion_expectation], timeout: defaultTimeout)
+        env.eventSender!.keystroke_completion_expectation = XCTestExpectation()
```

Comment:
Expectation can be waited only once, but this test wants to wait twice, so we set a new expectation.